recipes/pyglet/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name="pyglet" %}
+{% set version = "1.2.4" %}
+{% set md5="ba640b5fea8d670efdd747daa70a529a" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - pyglet
+    - pyglet.app
+    - pyglet.canvas
+    - pyglet.extlibs
+    - pyglet.libs
+    - pyglet.libs.x11  # [linux]
+    - pyglet.libs.darwin  # [osx]
+    - pyglet.libs.darwin.cocoapy  # [osx]
+    - pyglet.libs.win32  # [win]
+
+    # Skip all of the audio driver imports for now...
+
+    # skip to avoid installing audio drivers on Windows, see
+    # https://www.openal.org/platforms/
+    #- pyglet.media.drivers.openal  # [not win or linux]
+
+    # Again, skip the driver-specific import
+    #- pyglet.media.drivers.pulse  # [not win]
+
+    # Skip the carbon test because carbon isn't present on recent versions
+    # of OSX.
+    # - pyglet.window.carbon  # [osx]
+
+    - pyglet.window.win32  # [win]
+
+    # Would be nice to enable these eventually on non-Windows:
+    - pyglet.font  # [win]
+    - pyglet.gl  # [win]
+    - pyglet.graphics  # [win]
+    - pyglet.image  # [win]
+    - pyglet.image.codecs  # [win]
+    - pyglet.input  # [win]
+    - pyglet.media  # [win]
+    - pyglet.media.drivers  # [win]
+    - pyglet.media.drivers.directsound  # [win]
+    - pyglet.window  # [win]
+    - pyglet.text  # [win]
+    - pyglet.text.formats  # [win]
+
+about:
+    home: http://pyglet.readthedocs.org/en/pyglet-1.2-maintenance/
+    license: BSD 3-Clause License
```

Comment:
Normally we drop "License" too as it is already clear, but it is not a big deal either way.